pyk/src/pyk/utils.py

```diff
@@ -159,7 +163,20 @@ def shorten_hashes(value: Any, leftChars=6, rightChars=6) -> Any:
     return result
 
 
+def deconstruct_short_hash(h: str) -> Tuple[str, str]:
```

Comment:
Let's choose between `..` and `...` as separator.

I think the shorter the better.

Reply:
I think we do. What is the advantage of choosing `...` over `..`?

Reply:
Thought I saw `...` being used in the web interface, but think I was mistaken.